mandatory/1-fix-functions.js

```diff
@@ -32,7 +31,7 @@ function greaterThan10(num) {
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element = arr[index];
```

Comment:
Great! Hopefully indexing starting at 0 isn't that confusing for you, I know it took me a while to get used to it!